Core/Base/DivisaTools.php

```diff
@@ -49,4 +49,19 @@ public function format($number, $decimals = FS_NF0, $addSymbol = true)
 
         return $symbol . ' ' . $txt;
     }
+
+    /**
+     * Return format mask for edit grid
+     *
+     * @param type $decimals
+     * @return string
+     */
+    public static function gridMoneyFormat($decimals = FS_NF0)
```

Comment:
Probably for this is better use [str_pad](http://php.net/manual/en/function.str-pad.php), even better use [number_format()](http://php.net/manual/en/function.number-format.php) which support locales